vcpkg/users/platforms/android.md

```diff
@@ -40,24 +33,41 @@ There are four different Android ABIs, each of which maps to a vcpkg triplet. Th
 |---------------------------|----------------------|
 |arm64-android              | arm64-v8a            |
 |arm-android                | armeabi-v7a          |
+|arm-neon-android (w/neon)  | armeabi-v7a          |
```

Comment:
The lines above this say "There are four different Android ABIs,..." which seems incorrect now (6?).

Also, this table should have an extra column added to clarify the difference between `arm-android` and `arm-neon-android`. (`ANDROID_NEON_ENABLED`?)

Reply:
fixed!